OptimizelySDKCore/OptimizelySDKCore/OPTLYBuilder.m

```diff
@@ -87,7 +87,7 @@ - (NSData *)datafile {
 
 - (id<OPTLYEventDispatcher>)eventDispatcher {
     if (!_eventDispatcher) {
-        _eventDispatcher = [[OPTLYEventDispatcherNoOp alloc] init];
+        _eventDispatcher = [[OPTLYEventDispatcherBasic alloc] init];
```

Comment:
nit: can we rename this OPTLYEventDispatcherDefault to make the naming consistent with some of our other classes?